src/components/List.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import {Crypto} from './Interfaces';
+
+export const List = ({cryptos} /* :Crypto[] */) => {
+  let bull: boolean;
+
+  return (
+    <>
+      {cryptos.map((el: Crypto) => {
```

Comment:
move to a render function and use implicit return

Reply:
I don't know how to do that, I've investigated but I couldn't find a satisfactory information